.github/workflows/nightly.yml

```diff
@@ -142,6 +145,36 @@ jobs:
             name: wheels-${{ env.PY_VERSION }}-${{ matrix.cibw_arch }}
             path: ./dist/*.whl
 
+  test_wheels:
+    name: Test wheels
+    if: github.repository_owner == 'dipy' && github.ref == 'refs/heads/master'
+    needs: [build_linux_wheels]
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.12"]
+    steps:
+      - name: Rename Python version
+        shell: bash
+        run: echo "PY_VERSION=$(echo ${{ matrix.python-version }} | tr -d '.')" >> $GITHUB_ENV
+      - uses: actions/download-artifact@v4
+        with:
+          name: wheels-cp${{ env.PY_VERSION }}-manylinux2014-x86_64
+          path: ./dist
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Test wheel
+        run: |
+          set -eo pipefail
+          ls -al ./dist/*.whl
+          python -m pip install --only-binary="numpy,scipy,h5py" --index-url "https://pypi.anaconda.org/scientific-python-nightly-wheels/simple" "numpy>=2.1.0.dev0" "scipy>=1.14.0.dev0" h5py
+          python -m pip install ./dist/*.whl
+          python -c "import dipy; print(dipy.__version__)"
+          python -c "import numpy; assert int(numpy.__version__[0]) >= 2, numpy.__version__"
+          python -c "from dipy.align.imaffine import AffineMap"
+
```

Comment:
I will merge this for now but I believe this section is not needed.

we should use `CIBW_TEST_command` instead to be sure to test all wheels and not only one wheel (the linux one).

I will update that on another PR. 

Thank you very much for this work.